sidecar/config/config.go

```diff
@@ -146,6 +159,9 @@ type Config struct {
 	Supervise bool `yaml:"supervise"`
 
 	HealthChecks []HealthCheck `yaml:"healthchecks"`
+	HealthChecksCACertPath string `yaml:"healthchecks_ca_cert_path"`
```

Comment:
There is CACertPath in HealthChecks struct. This one is intended for ease of config via envir var and cmd line. The intent is to be similar to other health check stuff - you can specify the URL easily with A8_HEALTHCHECKS but for more detailed config have to use yaml. 

If I remove it we're basically saying health check tls must be configured in yaml...right?